### PR TITLE
docs: update Cloud onboarding docs

### PR DESCRIPTION
**Pull request by aaronsteers**
https://deploy-preview-7462--meltano.netlify.app/cloud/onboarding

### CODE REVIEW

File: docs/src/_cloud/onboarding.md [hunk 1]
```diff
@@ -52,44 +38,87 @@ Once you've completed the above, you are ready to onboard your project to Meltan
...
+To install from scratch using [pipx](https://pypa.github.io/pipx/installation/#install-pipx):
+
+```console
+pipx install 'git+https://github.com/meltano/meltano.git@cloud#subdirectory=src/cloud-cli'
+```
+
+To upgrade to the latest version:
+
+```console
+pipx reinstall meltano-cloud-cli
```
Comment (edgarrmondragon) [inline docs/src/_cloud/onboarding.md:63]: Does `pipx upgrade meltano-cloud-cli` not work with git-based installations?

Comment (aaronsteers) [reply docs/src/_cloud/onboarding.md:63]: More precisely, it becomes a no-op if there isn't a version bump.

File: docs/src/_cloud/encrypting_secrets.md [hunk 2]
```diff
@@ -32,21 +32,33 @@ meltano cloud config env delete <SECRET_NAME>
...
+
+See the [reserved variables](/platform/#reserved-variables) docs for more details on variables that are reserved for use by Meltano Cloud.
+
+## Alpha Phase Encryption Method (Deprecated)
+
+<div class="notification is-warning">
+  <p><strong>Meltano Cloud is currently in Beta.</strong></p>
+  <p>These Alpha instructions are now obsolete but are maintained for legacy
+  Alpha users.</p>
+</p>
```
Comment (WillDaSilva) [inline docs/src/_cloud/encrypting_secrets.md:45]:
```suggestion
```

**Review by WillDaSilva — DISMISSED** [review]

**Review by cjohnhanson — APPROVED** [review]
Looks good to me.

Unrelated to these changes, but all the "Additional Resources" links appear to be dead links. Is this intentional in the short term and we're planning to add those relatively soon? Do we want to get rid of those links for now and re-add once the linked content is up?

**Comment by aaronsteers** [comment]
> Looks good to me.
> 
> Unrelated to these changes, but all the "Additional Resources" links appear to be dead links. Is this intentional in the short term and we're planning to add those relatively soon? Do we want to get rid of those links for now and re-add once the linked content is up?

Good question. I couldn't find the issue but I don't want to block on fixing the stale content. We can log a follow-on issue to address the dead links.

**Review by WillDaSilva — APPROVED** [review]

**Pull request merged**